docs/getting-started/elements/html.md

```diff
@@ -1,8 +1,10 @@
-# Web Component
+# Elements in HTML
 
-Integrating Elements into any project is a case of loading up some JavaScript and CSS, and embedding a Web Component (custom HTML) into the HTML of whatever existing documentation you have. 
+Setting up Elements in HTML is a case of loading up some JavaScript and CSS and embedding a Web Component (custom HTML) into the HTML of your existing documentation. 
 
-This will work with pretty much any CMS: Drupal, Joomla, Wordpress, Jekyll, they will all allow custom HTML to be embedded, and this approach is supported by [pretty much all modern browsers](https://caniuse.com/custom-elementsv1).
+This works with pretty much any CMS: Drupal, Joomla, Wordpress, or Jekyll. They all allow custom HTML to be embedded and this approach is supported by [pretty much all modern browsers](https://caniuse.com/custom-elementsv1).
```

Comment:
> This works with pretty much any CMS - Drupal, Joomla, Wordpress, or Jekyll, etc.

Reply:
Removed it bdd9ff54b1a2434dc52989bae4f17c194597fc8d